pkg/scheduler/framework/cyclestate.go

```diff
@@ -58,6 +58,17 @@ type CycleState struct {
 	// the String set has been deprecated; transition to the generic set when the new version
 	// becomes available.
 	skippedFilterPlugins sets.String
+	// skippedScorePlugins is a set of Score plugins that should be skipped in the current scheduling cycle.
+	//
+	// TO-DO (chenyu1): the sets package has added support for Go generic types in 1.26, and
+	// the String set has been deprecated; transition to the generic set when the new version
+	// becomes available.
+	skippedScorePlugins sets.String
+	// desiredBatchSize is the desired batch size for the current scheduling cycle.
+	desiredBatchSize int
+	// batchSizeLimit is the limit on batch size for the current scheduling cycle, set by
+	// post-batch plugins.
+	batchSizeLimit int
```

Comment:
missed this comment before. add a comment here to indicate they're applied only for pickN type.